src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -1102,10 +1104,11 @@ public void StopAndAbort(SocketAsyncContext context)
 
                     if (_tail != null)
                     {
+                        aborted = true;
                         AsyncOperation op = _tail;
                         do
                         {
-                            op.TryCancel();
+                            aborted |= op.TryCancel();
```

Comment:
why do we need this? aborted was set to true above.

Reply:
that's a mistake, it should be set to true here.